ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml

```
@@ -304,6 +304,57 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )gke-postgres-scale-tests,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build01
+    context: ci/prow/gke-postgres-ui-e2e-tests
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+      timeout: 5h0m0s
+    labels:
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-stackrox-stackrox-master-gke-postgres-ui-e2e-tests
+    optional: true
+    rerun_command: /test gke-postgres-ui-e2e-tests
```

```suggestion
    rerun_command: /test gke-postgres-ui-e2e-tests
    run_if_changed: ^ui/
```
would match what's in `pull-ci-stackrox-stackrox-master-gke-ui-e2e-tests`.